public/stylesheets/site/2.0/02-elements.css

```diff
@@ -53,6 +53,8 @@ a:focus img {
 h1, h2, h3, h4, h5, h6, .heading {
   font-family: Georgia, serif;
   font-weight: 400;
+    word-wrap: break-word;
+
```

Comment:
Can you remove this extra blank line that snuck in? Then we can merge :tada:

Reply:
whoops 👀

all gone! ✌️